README.md

```diff
@@ -1,20 +1,26 @@
-# echidna
+# Echidna: A Fast Smart Contract Fuzzer <a href="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png"><img src="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png" width="75"/></a>
 
 ![Build Status](https://github.com/crytic/echidna/workflows/CI/badge.svg)
-![echidna logo](echidna.png)
 
 Echidna is a weird creature that eats bugs and is highly electrosensitive (with apologies to Jacob Stanley)
 
-More seriously, Echidna is a Haskell library designed for fuzzing/property-based testing of EVM code. It supports relatively sophisticated grammar-based fuzzing campaigns to falsify a variety of predicates.
+More seriously, Echidna is a Haskell package designed for fuzzing/property-based testing of Ethereum smarts contracts. It uses a relatively sophisticated grammar-based fuzzing campaigns based on a contract abi to falsify Solidity assertions or user-defined predicates.
```

Comment:
`s/abi/ABI/`
`s/package/program/`

maybe we also want to comment that echidna is extensible for your own niche use cases

Reply:
Done.